Jenkinsfile.treecompose

```diff
@@ -112,18 +108,18 @@ node(NODE) {
       // with Docker/OCI images.  Ideally there'd be a way to "bind mount" content
       // into the build environment rather than copying.
       stage("Prepare repo copy") { sh """
-          cp -a --reflink=auto * ${treecompose_workdir}/
-          repo=\$(readlink ${treecompose_workdir}/repo)
-          rm ${treecompose_workdir}/repo
-          cp -a --reflink=auto \${repo} ${treecompose_workdir}/repo
+          cp -a --reflink=auto * ${host_workspace}/
+          repo=\$(readlink ${host_workspace}/repo)
```

Comment:
Could `repo` here get renamed to something like `original_repo` to disambiguate things?